src/crimson/os/seastore/cached_extent.h

```diff
@@ -161,7 +253,12 @@ class CachedExtent : public boost::intrusive_ref_counter<
    * Implentation may use this call to fixup the buffer
    * with the newly available absolute get_paddr().
    */
-  virtual void on_initial_write() {}
+  void on_initial_write() {
+    touched_by = 0;
+    on_initial_commit();
```

Comment:
on_initial_commit() isn't a good name -- they happen at the same time so write vs commit is misleading.  How about on_initial_write_impl()?

Reply:
ok, will change it